src/core/js/views/pageView.js

```diff
@@ -22,32 +22,39 @@ define([
         },
 
         preRender: function() {
+            $.inview.lock('pageView');
             this.disableAnimation = Adapt.config.has('_disableAnimation') ? Adapt.config.get('_disableAnimation') : false;
             this.$el.css('opacity', 0);
             this.listenTo(this.model, 'change:_isReady', this.isReady);
         },
 
         isReady: function() {
-            if (this.model.get('_isReady')) {
-                _.defer(_.bind(function() {
-                    $('.loading').hide();
-                    $(window).scrollTop(0);
-                    Adapt.trigger('pageView:ready', this);
-                    var styleOptions = { opacity: 1 };
-                    if (this.disableAnimation) {
-                        this.$el.css(styleOptions);
-                        $.inview();
-                    } else {
-                        this.$el.velocity(styleOptions, {
-                            duration: 'fast',
-                            complete: function() {
-                                $.inview();
-                            }
-                        });
-                    }
-                    $(window).scroll();
-                }, this));
-            }
+            if (!this.model.get('_isReady')) return;
+
+            var performIsReady = function() {
+                $('.loading').hide();
+                $(window).scrollTop(0);
+                Adapt.trigger('pageView:ready', this);
+                $.inview.unlock('pageView');
+                var styleOptions = { opacity: 1 };
+                if (this.disableAnimation) {
+                    this.$el.css(styleOptions);
+                    $.inview();
```

Comment:
```suggestion
                    $.inview();
                    $(window).scroll();
                    return;
```
then you can lose the `else` block